src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), properties should be ordered alphabetically. Please rearrange the properties of the `.stars` selector.